wrappers/services.go

```diff
@@ -255,7 +249,7 @@ func StartServices(apps []*snap.AppInfo, disabledSvcs []string, flags *StartServ
 			for i := len(apps) - 1; i >= 0; i-- {
 				app := apps[i]
 				svc, activators := internal.SnapServiceUnits(app)
-				if e := stopService(systemSysd, cli, app.DaemonScope, append(activators, svc)); e != nil {
+				if e := stopService(systemSysd, cli, app.DaemonScope, append(activators, svc), false); e != nil {
```

Comment:
we have the usally problem with bool arguments, we either need to do something else or at least use a local const:

const disable = false

Reply:
Good point, thanks, changed this!